app/lib/mini-course.cjsx

```diff
@@ -17,81 +16,73 @@ module.exports = React.createClass
 
   statics:
     find: ({workflow}) ->
-      # Prefer fetching the tutorial for the workflow, if a workflow is given.
+      # Prefer fetching the tutorial for the workflow, so we know which one to fetch if multiple exist.
       if workflow?
         apiClient.type('tutorials').get workflow_id: workflow.id, kind: "mini-course"
           .then ([minicourse]) ->
             minicourse
```

Comment:
I added a conditional to display the input as a checkbox if there's only one and a radio if there are many to choose from in the workflow editor. I originally tried using radio buttons for all, but was getting these errors in the console when clicking the input: 
`Failed to load resource: the server responded with a status of 422 (Unprocessable Entity)`
`Uncaught (in promise) Error: PG::UniqueViolation: ERROR:  duplicate key value violates unique constraint "index_workflow_tutorials_on_workflow_id_and_tutorial_id"`